net.sourceforge.Pdfedit.json

```diff
@@ -0,0 +1,173 @@
+                        "fedora_rpm_patches/qt-x11-free-3.3.3-qembed.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.4-fullscreen.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.5-uic.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.6-qfontdatabase_x11.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.6-strip.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.7-arm.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.7-umask.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.8-odbc.patch"
+                    ]
+                },
+                {
+                    "type": "patch",
+                    "strip-components": 0,
+                    "options": ["-l"],
+                    "paths": [
+                        "fedora_rpm_patches/0079-compositing-types.patch",
+                        "fedora_rpm_patches/0080-net-wm-sync-request-2.patch",
+                        "fedora_rpm_patches/0084-compositing-properties.patch",
+                        "fedora_rpm_patches/qt-3.3.8-libpng15.patch"
+                    ]
+                },
+                {
+                    "type": "patch",
+                    "path": "qt3_flatpak_build_fixes.patch"
+                }
+            ]
+        },
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install --prefix=/app --with-system --with-random define=\"BOOST_SYSTEM_NO_DEPRECATED\" -sNO_BZIP2=1 cxxflags=-fPIC cflags=-fPIC"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.65.1/source/boost_1_65_1.tar.gz",
+                    "sha256": "a13de2c8fbad635e6ba9c8f8714a0e6b4264b60a29b964b940a22554705b6b60"
+                }
+            ]
+        },
+        {
+            "name": "pdfedit",
+            "buildsystem": "simple",
+            "build-commands": [
+                "QTDIR=/app autoconf",
+                "QTDIR=/app ./configure --prefix=/app --with-boost=/app",
+                "QTDIR=/app /usr/bin/make clean",
+                "mkdir /app/share || echo '/app/share already exists'",
+                "QTDIR=/app CFLAGS=\"$CFLAGS -w -fPIC\" CXXFLAGS=\"$CXXFLAGS -fPIC -w -DQSDEBUGGER -DQT_CLEAN_NAMESPACE -DCONFIG_DATA_PATH=/app/share\" /usr/bin/make -e",
+                "QTDIR=/app /usr/bin/make install"
+            ],
+            "post-install": [
+                "install -Dm644 /app/share/icon/pdfedit_icon.svg /app/share/icons/hicolor/scalable/apps/net.sourceforge.Pdfedit.svg",
+                "for size in 16 32 64 128; do
+                    rsvg-convert -w $size -h $size -f png -o $size.png /app/share/icons/hicolor/scalable/apps/net.sourceforge.Pdfedit.svg
```

Comment:
Unless it's used by the app, just SVG file is enough.

Reply:
Ok I just left the SVG.